roles/common/templates/rules.v4.j2

```diff
@@ -32,6 +32,7 @@ COMMIT
 {% if wireguard_enabled and wireguard_port|int == wireguard_port_avoid|int %}
 # Handle the special case of allowing access to WireGuard over an already used
 # port like 53
+-A PREROUTING -s {{ subnets|join(',') }} -p udp --dport {{ wireguard_port_avoid }} -j ACCEPT
```

Comment:
I think we can go with the only `-A PREROUTING ! -s {{ (subnets|cidr_merge)[0] }} --in-interface {{ ansible_default_ipv6['interface'] }} -p udp --dport {{ wireguard_port_avoid }} -j REDIRECT --to-port {{ wireguard_port_actual }}`

Reply:
I tested this and set `strongswan_network: 192.168.48.0/24` and it generated:
```
-A PREROUTING ! -s 10.19.49.0/24 --in-interface ens3 -p udp --dport 53 -j REDIRECT --to-port 51820
```
which wouldn't exclude the IPsec traffic. Of course most users won't change just one of the internal network ranges.

Reply:
Oh, indeed won't be merged. Then I suggest just replace ACCEPT to RETURN in the rule you added

Reply:
Good idea.